experimental/GModule/GaloisCohomology.jl

```diff
@@ -1398,14 +1398,12 @@ end
 
 function Base.show(io::IO, m::MIME"text/plain", a::RelativeBrauerGroupElem)
   io = pretty(io)
-  print(io, "Element of relative Brauer group of $(parent(a).k)\n")
-  ioC = IOContext(io, :supercompact => true, :compact => true)
+  print(io, "Element of relative Brauer group of ", Lowercase(), parent(a).k)
+  io = IOContext(io, :supercompact => true, :compact => true)
   print(io, Indent())
-  for (p,v) = a.data
-    show(ioC, p)
-    print(io, " -> ")
-    show(ioC, v)
-    print(io, "\n")
+  data = sort(a.data; by = x -> x isa AbsSimpleNumFieldEmbedding ? Inf : minimum(x))
+  for (p,v) in data
+    print(io, "\n", p, " -> ", v)
```

Comment:
Dictionaries are not sortable, you need to collect first
```suggestion
  data = sort(collect(a.data); by =(x -> first(x) isa AbsSimpleNumFieldEmbedding ? Inf : minimum(first(x))))
  for (p,v) in data
    print(io, "\n", p, " -> ", v)
```